autofit/non_linear/search/abstract_search.py

```diff
@@ -721,8 +722,19 @@ def result_via_completed_fit(
         model.freeze()
         samples = self.samples_from(model=model)
 
+        try:
+            search_internal = self.backend
+        except AttributeError:
+            search_internal = None
+
+        if search_internal is None:
+            try:
+                search_internal = self.paths.load_search_internal()
+            except FileNotFoundError:
+                search_internal = None

```

Comment:
Put this in the except

Reply:
Oh unless you expect self.backend to be None?

Reply:
Actually it's kind of odd to catch an attribute exception for an attribute of self